var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -40,7 +37,12 @@ class Pfunit(CMakePackage):
 
     depends_on('python@2.7:', type=('build', 'run'))  # python3 too!
     depends_on('mpi', when='+mpi')
+    depends_on('m4', when='@4.1.5:')
 
+    conflicts("%gcc@:8.3.9", when="@4.0.0:", msg='Older versions of GCC do '
+              'not support the Fortran 2008 features required by new pFUnit.')
+    # See https://github.com/Goddard-Fortran-Ecosystem/pFUnit/pull/179
+#    conflicts("+shared", when="@4.0.0:")
```

Comment:
Why is this commented out?  Should we add a comment above it explaining why?  Or fully remove it?  Or put it back in?

Reply:
Good catch.  I didn't mean to leave it commented out.